### PR TITLE
feature/bash

### PR DESCRIPTION
**Pull request by Mikyseu**

### CODE REVIEW

File: staff/miquel-guevara/bash/README.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
+# Bash
+
+Command-line interface commands typical in BASH
+
+## commit
+
+```sh
+$ commit
```
Comment (manuelbarzi) [inline staff/miquel-guevara/bash/README.md]: what is the commit command?

File: staff/miquel-guevara/bash/README.md [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+
+/archivo
+```
+
+## pwd
+
+```sh
+$ pwd
+
+/en que carpeta estamos
```
Comment (manuelbarzi) [inline staff/miquel-guevara/bash/README.md]: move comments out of code blocks

Comment (manuelbarzi) [reply staff/miquel-guevara/bash/README.md]: and add demo outputs

**Comment by heberjs** [comment]
add the code Task to your tasks.

File: staff/miquel-guevara/bash/README.md
Comment (fercostacastro) [inline staff/miquel-guevara/bash/README.md:1]: the explanations might be in English

File: staff/miquel-guevara/bash/README.md [hunk 3]
```diff
@@ -0,0 +1,247 @@
+# BASH
+
+Command-line interface commands typical in BASH
+
+## git commit
+
+```sh
+$ git commit
+
+captura una instantánea de los cambios preparados en ese momento del proyecto
```
Comment (Barrufat) [inline staff/miquel-guevara/bash/README.md]: Each command definition and examples should be written outside the back ticks 🧐

File: staff/miquel-guevara/bash/README.md [hunk 4]
```diff
@@ -0,0 +1,247 @@
...
+
+Ej: README.md  staff/
+```
+
+## ls -l
+
+```sh
+$ ls -l
+
+todas las carpetas pero mas info
```
Comment (manuelbarzi) [inline staff/miquel-guevara/bash/README.md]: move explanations out of code blocks

File: staff/miquel-guevara/bash/README.md
Comment (Mikyseu) [reply staff/miquel-guevara/bash/README.md:1]: It's done, thanks for your comment

File: staff/miquel-guevara/bash/README.md (hunk 1 above)
Comment (Mikyseu) [reply staff/miquel-guevara/bash/README.md]: Ya estaría corregido, gracias

File: staff/miquel-guevara/bash/README.md (hunk 2 above)
Comment (Mikyseu) [reply staff/miquel-guevara/bash/README.md]: Todo correcto ya, gracias

File: staff/miquel-guevara/bash/README.md (hunk 3 above)
Comment (Mikyseu) [reply staff/miquel-guevara/bash/README.md]: Tot ok ja, gracies

File: staff/miquel-guevara/bash/README.md (hunk 4 above)
Comment (Mikyseu) [reply staff/miquel-guevara/bash/README.md]: Ya están sacadas, muchas gracias

**Pull request merged**